CHANGELOG/CHANGELOG-1.1.md

```diff
@@ -0,0 +1,47 @@
+# v1.1.0 - Changelog since v1.0.0
+
+## Changes by Kind
+
+## Feature
+
+- Improved Windows Support
+  - Update driver to use CSI proxy beta for Windows ([#607](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/607), [@jingxu97](https://github.com/jingxu97))
+  - Add volume expansion support for Windows in GCE PD CSI driver ([#637](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/637), [@jingxu97](https://github.com/jingxu97))
+  - Add defensive check for Windows. GCE PD CSI driver only support ntfs for Windows. If other fstype is passed, return error. ([#641](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/641), [@jingxu97](https://github.com/jingxu97))
+  - Modify NodeUnstageVolume call for Windows ([#633](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/633), [@jingxu97](https://github.com/jingxu97))
```

Comment:
to accomplish what?

Reply:
to use csi_proxy dismount call which will include flush data cache before remove mount point.

Reply:
@jingxu97

Reply:
Updated doc.